language/en.json

```
@@ -16,6 +16,7 @@
     "semantics-form_save": "Save",
     "toolbar-button-type_map-color": "Map color",
     "toolbar-button-type_create-box": "Create box",
-    "toolbar-button-type_create-arrow": "Create arrow"
+    "toolbar-button-type_create-arrow": "Create arrow",
+    "toolbar-button-type_cannot-create-arrow": "There are no boxes to connect with the arrow"
```

Perhaps the error message should be more descriptive? We could tell the user how to fix the problem.

```suggestion
    "toolbar-button-type_cannot-create-arrow": "There must be at least two boxes before an arrow can be added"
```